frontend/src/components/organisms/ProfileForm.tsx

```diff
@@ -275,6 +403,11 @@ const ProfileForm = ({ userDetails }: ProfileFormProps) => {
               Reset changes
             </Button>
           </div>
+          <div className="order-2 sm:order-1">
+            <Button variety="error" type="button" onClick={handleOpen}>
+              Delete account
+            </Button>
+          </div>
```

Comment:
I saw some experimentation with making the Delete account button disabled until the user enters the right email. I think it's a good idea. Let's keep that